### ISSUE TITLE
Invalid backend response without another error message

### ISSUE THREAD

**Issue by DWE64** [helios-ag/FMElfinderBundle#503]
Good morning,
Here is the problem encountered: on a dockerized development environment (apache config with php 8.2) the bundle works well, I have correct access to a user's specific folder.
However, during the transfer via github of the project in preproduction, identical environment (and upload folder with permission 777 authorized at www-data:www-data for the tests), I no longer have access to the user's homeFolder.
Indeed, in the url called, the target parameter remains empty.
This returns me an invalid backend response with an empty but existing json data.
Note that this target parameter is not empty in the development environment. But it is managed by the bundle. So why is it empty in preproduction?

Here is the screenshot of the console, as well as the config file used. Only the URL is dynamic. Here in dev, FMELFINDER_URL = "http://localhost:8092/" and it works, while in pre-prod, FMELFINDER_URL="https://www.zonedev.bookrepsta.com/" and it doesn't work anymore. Moreover, no error log goes back to the server. Whether Apache logs or Symfony logs.

![image](https://github.com/helios-ag/FMElfinderBundle/assets/45743753/1e15c535-eb00-4457-9f9f-6bde67dbbcba)

```
fm_elfinder:
    assets_path: /assets
    instances:
        default:
            locale: '%locale%'
            editor: ckeditor
            #path_prefix: https://www.bookpresta.com/
            path_prefix: '%env(FMELFINDER_URL)%'
            fullscreen: true
            multi_home_folder: true
            folder_separator: "|"
            where_is_multi:
                roots: 0
            connector:
                debug: false
                roots:
                    upload:
                        #url : https://www.bookpresta.com/upload/package/{homeFolder}/
                        url : '%env(FMELFINDER_URL)%upload/package/{homeFolder}/'
                        tmb_url: '%env(FMELFINDER_URL)%upload/package/{homeFolder}/'
                        driver: LocalFileSystem
                        path: upload/package/
                        tmb_path: upload/package/.tmb/
                        upload_allow: ['image/png', 'image/jpg', 'image/jpeg']
                        upload_deny: ['all']
                        upload_max_size: 5M
                        attributes:
                            - {pattern: '..',read: true, write: true, locked: false}
```

Pre-production environment:
Ubuntu 20.04
Framework Symfony 6
Bundle Version: "helios-ag/fm-elfinder-bundle": "12.*"

thank you in advance for your help

**Comment by olia-bn** [comment]
[Enabling CORS support](https://github.com/helios-ag/FMElfinderBundle/blob/main/docs/cors-support.md) seems like a work-around.

Can we get some feedback on this issue? It affects `^12.3`.

**Comment by DWE64** [comment]
It worked. THANKS.
A feature that would be important to put would be a logger system to report errors... Because as soon as there is an error, debugging is done blind...
Anyway thanks for the help.

**Comment by bobvandevijver** [comment]
Fix has been released with 12.3.2.
